tests/emails/0.yml

```diff
@@ -0,0 +1,16 @@
+subject: 'zaÅ¼Ã³ÅÄ gÄÅlÄ jaÅºÅ i zrÃ³b prÃ³bne zadanie %%tod'
+from:
+  - name: Pawel
+    email: pawel@test.com
+to:
+  - name: ''
+    email: dan@test.com
+cc: []
+bcc: []
+sender: []
+plainbody: |
+            This is a nonesense test-email to check Plancake Mail to Inbox feature. 
+            All Polish diacritics included ;-) tag also included for additional tests.
+            It should look like in the picture attached.
+            Best regards
+            Pawel
```

Comment:
Can you please ensure that all of these files have a trailing newline.
See here for context: https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file